contributing/releasing.md

```diff
@@ -13,10 +13,19 @@ Each release is driven by a single **release engineer**, who is also a Googler.
 is expected to do the following, in order of priority:
 
 - Do not break Google.
+- Cut a release every Wednesday morning before 10am EST, without exception.
```

Comment:
Since I don't think we require anyone to actually have arrived at work by 10am Eastern Time, should we push this out to 12pm?

Also, can we specify "New York time" or "Eastern Time (US and Canada)"? Choosing Eastern Standard Time may cause confusion during the Daylight Saving Time months.

Reply:
This isn't an implication of a requirement that anyone shows up before 10am - it's fair and reasonable to cut the release Tuesday evening as well prior to leaving work. The important fact here is that we do not let the cutting of the release slip due to waiting for a feature to land.

Reply:
Is that fair to authors of PRs who are trying to "make the deadline"? If the release cop decides to cut the release at 3:00pm on a Tuesday, others hoping to land a PR around 5pm would have to wait one more week.  This is assuming it's not a critical issue warranting a Hotfix release.

Reply:
Likely not fair, no, but I think this will have to come down to collaboration with the release engineer for a given week. The harsh truth is that if it doesn’t make the cut, it doesn’t make the cut. This fact is in fairness to all the other changes that did make it into the release.

Reply:
I think this thread isn't worth the engineering time—too small to matter.

I guess one could say "first thing Wednesday morning, NYC time" if you wanted to avoid encoding a specific time, but I defer to Jeff here.

The idea with a regular, predictable release cut is that features that miss one release will just roll into the next—you don't have to wait a long time.